home/ivp-planning/getting-started/importing-pdfs.md

```diff
@@ -1,31 +1,21 @@
 # Importing PDFs
 
-In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.
+To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.
 
+![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)
 
-{% embed url="https://youtu.be/MuLt94b64O8?t=203" %}
-How to import and export PDFs (german)
-{% endembed %}
+Before iVP actually processes the PDF and imports it, you will be asked to set some parameters that specify how it will be loaded. The options are:
 
-## Importing PDFs step by step:
+* **Page Size**: Changes the size of the imported PDF, you can either choose from a preset or manually customize the size of the file. Additionally, you can use the dimensions saved in the PDF.
 
-To import a PDF click on **File -> PDF -> Import PDF**. A new window will open that allows you to navigate to the folder that contains the PDF you want to import.
+* **Width Original File**: Displays the original files width, which can be adjusted if the file size isn't taken from it. Also changes the overall width and scale of the PDF when modified.
```

Comment:
"Displays the original files width, which can be adjusted if the file size isn't taken from it." This sounds a bit clunky IMO